src/views/landing.njk

```diff
@@ -7,36 +7,36 @@
   {{ serviceName }}
 {% endblock %}
 
-{% block beforeContent %}
-  {{
-    govukBreadcrumbs({
-      items: [
-        { href: 'https://www.gov.uk/', text: 'Home' },
-        { href: 'https://www.gov.uk/browse/business', text: 'Business and self-employed' },
-        { href: 'https://www.gov.uk/browse/business/limited-company', text: 'Running a limited company' }
-      ]
-    })
-  }}
-{% endblock %}
-
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
-      <p class="govuk-body-l">Use this service to get your NodeJS application up and running</p>
-      <p>An example form submission with validation is included</p>
-      <form method="post">
-        {{
-          govukButton({
-            text: 'Start now',
-            isStartButton: true,
-            attributes: {
-              id: 'start-now',
-              onclick: '_paq.push(["trackGoal", 1]);' if piwik
-            }
-          })
-        }}
-      </form>
-    </div>
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+       <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
+        <p class="govuk-body-l">Use this service to apply to close a company, also known as striking a company off the register or dissolution.</p>
+        <p class="govuk-body">It costs £8 to close a company and you'll need:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the company number</li>
+          <li>a Companies House password - you'll be able to register if you don't have one</li>
+          <li>the company authentication code</li>
+          <li>an email address for each director if the company has more than one</li>
+          <li>a credit or debit card</li>
+        </ul>
+        <form method="post">
+         {{
+            govukButton({
+              text: 'Start now',
+              href: "/who-to-tell",
```

Comment:
wait, how does this work? Does it call the POST method in the landing controller, or navigate directly to the Who to tell page? If it calls the controller, then remove the href here.

I'm surprised the href works too. Would've thought it needed to be `/close-a-company/who-to-tell`